src/App.tsx

```
@@ -1,16 +1,19 @@
 import { useState } from 'react';
 import './App.css';
+import Markdown from 'react-markdown';
 import Editor from './components/Editor';
 import Preview from './components/Preview';
 
+// import {mockMarkdown} from './mocks/mock'
+
 function App() {
   const [markdownInput, setMarkdownInput] = useState('');
```

ESLint

[error] 'setMarkdownInput' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)